src/System.Text.Formatting/System/Text/Formatting/IOutputExtensions.cs

```diff
@@ -12,13 +12,13 @@ public static class IOutputExtensions
         public static void Append<TFormatter, T>(this TFormatter formatter, T value, SymbolTable symbolTable, StandardFormat format = default) where T : IBufferFormattable where TFormatter : IOutput
         {
             while (!formatter.TryAppend(value, symbolTable, format)) {
-                formatter.Enlarge();
+                formatter.GetMemory();
             }
         }
 
         public static bool TryAppend<TFormatter, T>(this TFormatter formatter, T value, SymbolTable symbolTable, StandardFormat format = default) where T : IBufferFormattable where TFormatter : IOutput
         {
-            if (!value.TryFormat(formatter.GetSpan(), out int bytesWritten, format, symbolTable))
+            if (!value.TryFormat(formatter.GetMemory().Span, out int bytesWritten, format, symbolTable))
```

Comment:
These .Span calls are annoying.

Reply:
There would be no .Span class if we move to OutputWriter